meta-mender-core/classes/mender-setup.bbclass

```diff
@@ -244,10 +244,10 @@ python() {
 addhandler mender_sanity_handler
 mender_sanity_handler[eventmask] = "bb.event.ParseCompleted"
 python mender_sanity_handler() {
-    if bb.utils.contains('MENDER_FEATURES_ENABLE', 'mender-partuuid', True, False, d) and d.getVar('MENDER_STORAGE_DEVICE', True) != "":
+    if bb.utils.contains('MENDER_FEATURES', 'mender-partuuid', True, False, d) and d.getVar('MENDER_STORAGE_DEVICE', True) != "":
         bb.warn("MENDER_STORAGE_DEVICE is ignored when mender-partuuid is enabled. Clear MENDER_STORAGE_DEVICE to remove this warning.")
 
-    if bb.utils.contains('MENDER_FEATURES_ENABLE', 'mender-partuuid', True, False, d) and bb.utils.contains('MENDER_FEATURES_ENABLE', 'mender-uboot', True, False, d):
+    if bb.utils.contains('MENDER_FEATURES', 'mender-partuuid', True, False, d) and bb.utils.contains('MENDER_FEATURES', 'mender-uboot', True, False, d):
```

Comment:
Careful! `MENDER_FEATURES_ENABLE` end up in `DISTRO_FEATURES` in dunfell and older. From grepping I can see that there are already a couple of mistakes in dunfell regarding this, notably when handling `mender-partuuid`.

Reply:
Thanks.

Fixed this on the cherry-pick and also added a [separate commit for the existing broken feature](https://github.com/mendersoftware/meta-mender/pull/2125/commits/95f38e932a3e44b44b9d357f705ac087d617a765). Can you please give feedback on that commit? Is the changelog too much and we could do `Title` instead?

Reply:
I think I'd go for "title", the full message is a bit much IMHO. Looks good otherwise!